data/humans/occupations.json

```
@@ -942,6 +946,7 @@
 		"upholsterer",
 		"urban planner",
 		"usher",
+    "ux designer",
```

Shall we put this uppercase for clarity? People can convert to lower case later if they like. 
```suggestion
    "UX designer",
```